changelogs/fragments/70319-reduce-ignored-module-sanity-tests.yml

```
@@ -1,2 +1,4 @@
 bugfixes:
   - apt_repository - fixes ``mode`` doc to remove ineffective default (https://github.com/ansible/ansible/pull/70319).
+deprecated_features:
+  - apt_key - The paramater ``key`` does not have any effect, has been deprecated and will be removed in ansible version 2.13.
```

```suggestion
  - apt_key - the paramater ``key`` does not have any effect, has been deprecated and will be removed in ansible-base version 2.14 (https://github.com/ansible/ansible/pull/70319).
```
The deprecation cycle is four versions (changes in devel will end up in 2.11, so this will get removed in 2.14).